src/views/TestDate/UpdateTestDate.tsx

```diff
@@ -50,16 +54,12 @@ const TestDate = () => {
     date => date.id === testDateId
   );
 
-  if (!test) {
-    throw new TypeError('TypeError: test does not exist');
```

Comment:
I wasn't sure why we were throwing an error here.  I opted to send someone to the 'Not found' view if the expected test didn't exist on the request.